tests/src/whisk/core/controller/test/SequenceApiTests.scala

```diff
@@ -300,19 +294,11 @@ class SequenceApiTests
         val sSeq = makeSimpleSequence(sSeqName, namespace, Vector(xSeqName, xSeqName), false) // x is installed in the db already
         val content = WhiskActionPut(Some(sSeq.exec))
 
-        implicit val stream = new java.io.ByteArrayOutputStream
-        this.outputStream = new PrintStream(stream)
-        try {
-            stream.reset()
-            Console.withOut(stream) {
-                Put(s"$collectionPath/$sSeqName", content) ~> sealRoute(routes(creds)) ~> check {
-                    status should be(OK)
-                    logContains(s"atomic action count ${2 * actionCnt}")
-                }
+        Console.withOut(stream) {
+            Put(s"$collectionPath/$sSeqName", content) ~> sealRoute(routes(creds)) ~> check {
+                status should be(OK)
+                logContains(s"atomic action count ${2 * actionCnt}")(stream)
```

Comment:
@ioana-blue we really should refactor the code so that you can check these invariants by calling the methods themselves rather than checking the logs.